src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -47,7 +47,15 @@ export const useTradeExecution = (
   const trackMixpanelEvent = useMixpanel()
   const hasMixpanelSuccessOrFailFiredRef = useRef(false)
 
-  const sellAssetAccountId = useAppSelector(state => selectHopSellAccountId(state, hopIndex))
+  const selectHopSellAccountIdFilter = useMemo(() => {
```

Comment:
nitpick: remove `select` terminology which implies this is a selector

```suggestion
  const hopSellAccountIdFilter = useMemo(() => {
```

Reply:
Good catch! Addressed all instances in https://github.com/shapeshift/web/pull/7560/commits/20407119916771f67359dadcae5ee3c6d42c5b44